public/manifest.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
FYI irrelevant to this PR and you may already know, but NextJS has integrated manifest support:

https://nextjs.org/docs/app/api-reference/file-conventions/metadata/manifest

Don't personally see much of a difference doing it this way vs that way, but I'm not an expert on PWAs anyway